scripts/template_openxr_flags.hpp

```diff
@@ -28,6 +28,14 @@ enum class /*{projected_bits_type }*/ : XrFlags64 {
 
 using /*{projected_type }*/ = Flags</*{projected_bits_type }*/, /*{flags.name}*/>;
 
+OPENXR_HPP_INLINE /*{projected_type }*/ operator|( /*{projected_bits_type }*/ bit0, /*{projected_bits_type }*/ bit1 ) {
+    return /*{projected_type }*/( bit0 ) | bit1;
+}
+
+OPENXR_HPP_INLINE /*{projected_type }*/ operator~( /*{projected_bits_type }*/ bits ) {
+    return ~( /*{projected_bits_type }*/( bits ) );
```

Comment:
this looks like it's constructor-casting the bits type to itself... maybe this is what you meant instead?

```suggestion
    return ~( /*{projected_type }*/( bits ) );
```